server/Main.hs

```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PatternGuards #-}
```

Comment:
I'm not sure why this has started happening, but CI fails on the server with:

```
[1 of 1] Compiling Main

/home/travis/build/purescript/trypurescript/server/Main.hs:136:27: error: [-Werror]

    accepting non-standard pattern guards (use PatternGuards to suppress this message)

        Just js <- codegenResult

    |

136 |                   Right _ | Just js <- codegenResult -> do

    |      
```

Reply:
Maybe we updated to a newer GHC version which produces this warning where it didn't previously?